package.json

```diff
@@ -34,20 +34,25 @@
   },
   "homepage": "https://github.com/nhsuk/profiles#readme",
   "dependencies": {
+    "after-brunch": "^0.0.5",
     "body-parser": "^1.18.2",
+    "brunch": "^2.10.12",
```

Comment:
There is still the problem that Brunch (et al) is a production dependency and should be a devDependency. This is also a problem in the nhsuk-prototype-toolkit and requires some thought to resolve. One solution would be to move them into devDependencies and then install & remove the devDependencies as part of the build.

See https://github.com/nhsuk/profiles/pull/261 as an example of how that might work.

It does also raise the question of what NODE_ENV should be on CI. I don't think it is either production or development.

Reply:
Alex and I discussed this and I’d argue that it is a production dependency. It’s used to generate assets that are only used in production and this is the only reason it is in the project. That’s the logic anyway.
The env on CI is a good thing to question and maybe it should change depending on what stage it is running. However, during the stage of creation the image for the application IMO production is correct. I wouldn’t want the application running in development.
Probably something worth talking through...

Reply:
On reflection, I think my focus on _is brunch a production or dev dependency_ misses the point. I think the real issue is should brunch be deployed to production (which is tangentially related but not the same thing).

Reply:
I'm confused as to how the approach in 261 works - is this a fair summary:

brunch has moved to dev dependencies
the docker build uses a `production=false` switch to override the production environment setting so that the brunch dev dependencies is available. This means all dev dependencies will be installed.
When the app runs the production false switch is not there, so the app runs as production, however all the dev dependencies will be there as the yarn installed as development.

If that's what's happening, I'd say it's better to have brunch in the production dependency list, then at least only the brunch dependencies are deployed, not all the other dev dependencies.

Unless brunch is moved out of the container and the sccs files built before the docker build I don't see how we can avoid brunch being deployed.